.github/workflows/ai-review.yml

```
@@ -0,0 +1,138 @@
+          # ------------------------------------------------------------------
+          # 2. Truncate diff at 40 KB
+          # ------------------------------------------------------------------
+          MAX_BYTES=40960
+          DIFF=$(echo "$DIFF" | head -c "$MAX_BYTES")
+
+          # ------------------------------------------------------------------
+          # 3. Call GLM-5.1 with retries + exponential back-off
+          # ------------------------------------------------------------------
+          SYSTEM_PROMPT='You are an expert code reviewer for a children'\''s reading application. \
+Review the following diff for: security (especially child data protection and COPPA compliance), \
+correctness, error handling, race conditions, test coverage, and performance. \
```

actionlint (1.7.12)

[error] could not parse as YAML: could not find expected ':'

(syntax-check)